src/homework13/BookManager.java

```diff
@@ -0,0 +1,86 @@
+package homework13;
+
+import java.io.*;
+import java.util.*;
+
+public class BookManager {
+
+    Map<String, Integer> words = new HashMap<>();
+    List<Words> sortedWords = new ArrayList<>();
+    final int SELECT_TOP_N = 10;
+
+    private String bookName;
+
+    public String getBookName() {
+        return bookName;
+    }
+
+    public void setBookName(String bookName) {
+        this.bookName = bookName;
+    }
+
+    public boolean findBook(String name) {
```

Comment:
name needs to be descriptive, like processBook or something like that, this method not only find book in also counts the words